src/transmogrify/dexterity/tests/schemaupdater.txt

```diff
@@ -99,8 +98,8 @@ DublinCore fields populated:
     True
     >>> spam.creators
     ('user1',)
-    >>> spam.effective_date
-    DateTime('2020/07/10 10:24:00 UTC')
+    >>> spam.effective_date.year(), spam.effective_date.month(), spam.effective_date.day()
+    (2020, 7, 10)
```

Comment:
I think you had to do this, because you removed the `TZ` environment variable, which was set in the buildout:

https://github.com/collective/transmogrify.dexterity/blob/1c761697928ba62acf18b6c1a69d1207e0015075/base.cfg#L55-L57

Maybe it works, if you do it in `testing.py` file.

Reply:
That is probably the reason. Thanks for spotting that